Makefile

```diff
@@ -73,6 +74,40 @@ lint:
 mypy:
 	pip install tox
 	tox -e mypy
+# todo: `mypy-simple` issues: (i) run against which dirs? `sssom/` and `tests/` only?,
+# ...(ii) it doesn't recognize dependencies.
+#mypy-simple:
+#	pip install mypy
+#	mypy
+
+# todo: see (i)
+.PHONY: qc-manifest
+qc-manifest:
+	tox -e manifest
+
+.PHONY: qc-flake8 qc-flake8-simple
+qc-flake8:
+	tox -e flake8
+qc-flake8-simple:
+	pip install flake8
+	tox -e flake8
+
+.PHONY: qc-pytest qc-pytest-simple
+qc-pytest:
+	tox -e py
+qc-pytest-simple:
+	python -m unittest discover
```

Comment:
Here, `tox -e py` is all you need to run `pytest` if you intend to run tests. The `qc-pytest-simple` may seem redundant

Reply:
I know we discussed on Slack already, but just replying here anyway. Running some of the the `tox -e` commands ended up deleting my environment for some reason. Maybe I'll figure out why eventually. For now, I'll just stick with running all of them via `tox`.